feed/views.py

```diff
@@ -1,60 +1,184 @@
+                image_data.append({i : item.image_url})
+                i += 1
+
+            return JsonResponse({
+                'current_user_id'   : get_current_user_id(request),
+                'feed_basic_data'   : {
+                    'feed_id'           : feed_id,
+                    'feed_user'         : feed_writer.nickname,
+                    'feed_user_id'      : feed_writer.id,
+                    'feed_writer_about' : feed_writer.about,
+                    'created_at'        : feed_data.created_at,
+                    'description'       : feed_data.description,
+                    'like_number'       : feed_data.like_number,
+                    'tag_item_number'   : feed_data.tag_item_number,
+                },
+                'product_data'      : product_data,
+                'feed_comment_data' : {
+                    'feed_comment_count' : feed_data.comment_set.count(),
+                    'comment_list'       : comment_list,
+                },
+                'feed_image_data'   : image_data,
+            }, status=200)
+
+        except ValueError:
+            return JsonResponse({'MESSAGE' : 'INVALID_VALUE_TYPE'}, status=400)
+
+        except KeyError:
+            return JsonResponse({'MESSAGE' : 'KEY_ERROR'}, status=400)
+
+        except Feed.DoesNotExist:
+            return JsonResponse({'MESSAGE' : 'INVALID_FEED_ID'}, status=404)
+
+        except Product.DoesNotExist:
+            return JsonResponse({'MESSAGE' : 'INVALID_PRODUCT_ID'}, status=404)
+
+    @login_decorator
+    def patch(self, request, feed_id):
+        try:
+            target_feed = Feed.objects.get(id=feed_id)
+            
+            if target_feed.user_id == request.user.id:
+                new_description = json.loads(request.body)['description']
+                if not new_description:
+                    return JsonResponse({'MESSAGE' : 'NO_FEED_DESCRIPTION'}, status=400)
+
+                current_feed = Feed.objects.get(id=feed_id)
+                current_feed.description = new_description
+                current_feed.save()
+                
+                return JsonResponse({'MESSAGE' : 'FEED_DESCRIPTION_UPDATED'}, status=200)
+            
+            return JsonResponse({'MESSAGE' : 'USER_WITHOUT_AUTHORITY'}, status=403)
+
+        except ValueError:
+            return JsonResponse({'MESSAGE' : 'INVALID_VALUE_TYPE'}, status=400)
+
+        except KeyError:
+            return JsonResponse({'MESSAGE' : 'KEY_ERROR'}, status=400)
+
+        except Feed.DoesNotExist:
+            return JsonResponse({'MESSAGE' : 'INVALID_FEED_ID'}, status=404)
+
+    @login_decorator
+    def delete(self, request, feed_id):
+        try:
+            target_feed = Feed.objects.get(id=feed_id)
+            
+            if target_feed.user_id == get_current_user_id(request):
+                Feed.objects.get(id=feed_id).delete()
+                
+                return JsonResponse({'MESSAGE' : 'FEED_DELETED'}, status=200)
```

Comment:
```suggestion
                return JsonResponse({'MESSAGE' : 'FEED_DELETED'}, status=204)
```
자원이 삭제 되었을 때에는 `no content`를 의미하는 204 코드를 추천드립니다

Reply:
감사합니다 수정했습니다.